system/blueprints/config/system.yaml

```diff
@@ -430,6 +430,16 @@ form:
                         0: PLUGIN_ADMIN.NO
                     validate:
                         type: bool
+                pages.markdown.extra_escape_fences:
+                    type: toggle
+                    label: Escape HTML elements in markdown extra fences
```

Comment:
This label should also be a lang string, and should be added to admin's `en.yaml` lang file.